internals/feature_helpers.py

```diff
@@ -61,7 +95,7 @@ def get_by_ids(feature_ids: list[int],
       if feature_id in result_dict]
   return result_list
 
-def get_all(limit=None, order='-updated', filterby=None,
+def get_all_legacy(limit=None, order='-updated', filterby=None,
```

Comment:
This can be for a follow-up PR: I don't see any callers of `get_all()` or `get_all_legacy()` that pass `keys_only`, so support for that could probably be deleted.